scripts/hv_stats/generate_bokeh_plots.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
So this file runs a python script that autogenerates the content for statistics/bokeh and then moves it into the web directory. I think everything in statistics/bokeh should not be in the main repo, so remove them from the PR.

Reply:
Ok will remove.